frontend/src/libs/helpers/check-subscription/check-subscription.ts

```diff
@@ -0,0 +1,10 @@
+const checkSubscription = (start: Date, end: Date): boolean => {
```

Comment:
```suggestion
const checkHasSubscription = (start: Date, end: Date): boolean => {
```
don't forget about file name